lib/s3/serverless/dispatcher.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+
+const S3ServerlessHandlerError = require('./error');
+
+const isObject = value => typeof value === 'object' && !Array.isArray(value);
+
+class ServerlessDispatcher {
+
+	/**
+	 * Dispatch the event to the listener package
+	 *
+	 * @static
+	 * @param {*} Listener
+	 * @param {*} event
+	 * @memberof ServerlessDispatcher
+	 */
+	static async dispatch(Listener, event) {
+
+		if(typeof event !== 'object' || !Object.keys(event).length)
```

Comment:
:thinking:  Talvés se podría aprovechar la función que valida que sea un objeto

```suggestion
		if(!isObject(event) || !Object.keys(event).length)
```